server/routes/common/api.ts

```diff
@@ -28,10 +34,11 @@ export default class CommonApiRoutes {
       })
       .catch(error => logger.error(error))
 
-    if (prisonerNumber === 'placeholder') {
+    const { placeholder } = this.photoService.getPhotoStatus(prisonerData, inmateDetail, alertSummaryData)
+
+    // If there's no photo ID then we dont need to call the API and can prevent the extra call
+    if (placeholder) {
       res.redirect(placeHolderImage)
-    } else if (prisonerNumber === 'photoWithheld') {
```

Comment:
This was never used as it came from the `photoType` value that would only be shown if the prisoner number wasn't defined